templates/app/commercial-vendor-program.phtml

```diff
@@ -0,0 +1,78 @@
+        <div class="col">
+            <h1>Commercial Vendor Program</h1>
+        </div>
+    </div>
+    <div class="row align-items-start mt-5">
+        <div class="col-12">
+            <h2>Existing Members</h2>
+        </div>
+    </div>
+
+    <div class="card-deck">
+    <?php foreach($commercialVendors as $commercialVendor): ?>
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title"><a target="_blank" href="<?= $this->escapeHtmlAttr($commercialVendor['url']) ?>"><?= $this->escapeHtml($commercialVendor['name']) ?></a></h5>
+                <p class="card-text"><?= $this->escapeHtml($commercialVendor['text']) ?></p>
+            </div>
+            <img class="card-img-bottom px-5 pb-2" src="<?= $this->escapeHtmlAttr($commercialVendor['logo']) ?>" alt="<?= $this->escapeHtmlAttr($commercialVendor['name']) ?>">
+            <div class="card-footer">
+                <a class="card-link" target="_blank" href="<?= $this->escapeHtmlAttr($commercialVendor['url']) ?>">Visit <?= $this->escapeHtml($commercialVendor['name']) ?></a>
+            </div>
+        </div>
+    <?php endforeach ?>
+    </div>
+
+    <h2 class="mt-5">Benefits</h2>
+    <p>
+        When you join the Commercial Vendor Program, you will be recognised as one of the key supporters of
+        Laminas. You will be helping to fund and ensure the success of the project.
+    </p>
+
+    <h2>About the Program</h2>
+    <p>
+        Your organisation can join the Commercial Vendor Program to help fund the ongoing maintenance and
+        development costs of the Laminas Projects.
+    </p>
+
+    <ul>
+        <li>
+            The cost is currently $1,000 per year (USD), with an initial 3 year minimum commitment. After the
+            initial commitment, the membership can be renewed year-on-year as agreed.
+        </li>
+        <li>
+            You agree to abide by the
+            <a href="https://github.com/laminas/.github/blob/master/CODE_OF_CONDUCT.md">Code of Conduct</a>,
+            and most importantly, that you shall not bring the Laminas Projects into disrepute.
+        </li>
+        <li>
+            On application to join the Commercial Vendor Program, your entry will be considered and voted on
+            by the <a href="<?= $this->url('about.tsc') ?>">Technical Steering Committee</a> (TSC).
+            Additionally, the TSC reserves the right to vote on whether a vendor should remain part of the
+            program (for example, if they have brought Laminas Project into disrepute).
+        </li>
+    </ul>
+
+    <h2>How to Apply</h2>
+    <ol>
+        <li>
+            First, contact <a href="mailto:<?= CVP_CONTACT_EMAIL ?>"><?= CVP_CONTACT_EMAIL ?></a> to apply for membership. Your
+            application will be considered and voted on by the TSC.
+        </li>
+        <li>
+            Once the TSC has approved your membership request, you must then make the required donation on the
```

Comment:
As the list started off with "First", I'd suggest continuing the list in that way, i.e., "Second" & "Third", etc or "Then" and "Finally".